README.md

```diff
@@ -23,7 +23,7 @@ And of course, we try to respond in a timely manner to GitHub issues, pull reque
 ## Developing
 
 We're glad you want to help! We have a **[handy guide for
-contributors](/doc/CONTRIBUTING.md)**. It can hopefully point you to helpful
+contributors](/.github/CONTRIBUTING.md)**. It can hopefully point you to helpful
```

Comment:
Would you be friendly to moving it to the root instead? That directory is getting crowded, but I think this guide is useful enough to go there. I'd like it to be easily noticed by people skimming the project.